lzma-conduit.cabal

```diff
@@ -29,9 +29,9 @@ library
   build-depends:
     base              >= 3      && < 5,
     bytestring        >= 0.9.1  && < 0.11,
-    conduit           >= 1.0    && < 1.3,
+    conduit           >= 1.0    && < 1.4,
```

Comment:
btw, I wasn't able to reach any `conduit == 1.0.*` version with the cabal solver; it makes more sense to `>= 1.1.0`

Reply:
Ok, I increased it.